src/ts/components/subtitleoverlay.ts

```diff
@@ -86,6 +86,20 @@ export class SubtitleOverlay extends Container<SubtitleOverlayConfig> {
 
       this.applyConfToDom(labelToAdd.getDomElement());
 
+      // In this block all the logic for CEA-608 captions is applied
+      // It's based on the assertion that a cue with position metadata
+      // is CEA-608
+      if (event.position != null) {
+        let domElement = labelToAdd.getDomElement();
+        domElement.css('position', 'fixed');
```

Comment:
Ok, I guess the lest and top property still need to be added directly as CSS, I don't think creating 31 classes is a good idea.